plans/example.pp

```diff
@@ -1,31 +1,31 @@
-plan snow_record::example(
+plan servicenow_tasks::example(
```

Comment:
Is this file necessary? I think we can assume that people using this module have a pretty good idea of how to write a plan using these tasks (that's unless `plans/example.pp` is a common convention for multiple modules, of which I don't know the answer to).

Reply:
I think it's common to have examples, but typically they are in an examples directory.

 examples/
    Contains examples showing how to declare the module's classes and defined types.
    init.pp: The main class of the module.
    example.pp: Provide examples for major use cases.

Reply:
Cool. Let's do that then?

Reply:
I moved it to examples, but you can't actually run it from there. Also it breaks rubocop. Moved it back to plans.

Reply:
That's annoying because this will show up in `bolt plan show`. Could we maybe add this plan to the README instead, like under an `Example Plan` section?

Reply:
That seems cluttery too. I think your original instinct was right. This doesn't need to be the place to look for how to write a plan. I'm just deleting it.